bin/grep

```diff
@@ -199,8 +199,8 @@ sub parse_args {
 		else {                # if use of Term::Cap fails,
 			my $term = $ENV{'TERM'} || 'vt100';
 			( $SO, $SE ) = $opt{'g'}    # use tput command to get escapes
-				? ( `tput -T $term smso`, `tput -T $term rmso` )
-				: ( `tput -T $term smul`, `tput -T $term rmul` );
+				? ( `tput -T '$term' smso`, `tput -T '$term' rmso` )
```

Comment:
wouldn't we still be able to break this with single ticks in TERM, like `TERM="vt100' ; rm -rf / '"` (or something close)?

Maybe there are still some of the exploits I show in https://www.perl.com/article/quoting-the-shell/

Reply:
Good point; I will add a new commit